ec2utils/ec2uploadimg/lib/ec2utils/ec2uploadimg.py

```diff
@@ -53,6 +56,7 @@ def __init__(self,
                  ssh_key_pair_name=None,
                  ssh_key_private_key_file=None,
                  ssh_timeout=300,
+                 temporary_key_created=False,
```

Comment:
This is only used internally so I wouldn't expose it to the constructor parameters

Reply:
Right. Thanks!